# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1242588894 -7200
#      Sun May 17 21:34:54 2009 +0200
# Node ID cb163402bf794c145f5bbbf3a0adb39137b9164d
# Parent  25f50d2d76b3e4b9a8b371546c711134fcd9a54b
narrow include in Array-tc.cc

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2009-05-17  Jaroslav Hajek  <highegg@gmail.com>
+
+	* TEMPLATE-INST/Array-tc.cc: Change #include.
+
 2009-05-15  Robert T. Short  <octave@phaselockedsystems.com>
 
 	* ov-class.cc (octave_class::exemplar_info): Fix typo in error message.
 	* ov-class.cc (Fmethods): Methods returns cell array.
 	* ls-mat5.cc (read_mat5_binary_element):
 	Construct exemplar array and ensure inheritance is correct.
 
 2009-05-15  John W. Eaton  <jwe@octave.org>
diff --git a/src/TEMPLATE-INST/Array-tc.cc b/src/TEMPLATE-INST/Array-tc.cc
--- a/src/TEMPLATE-INST/Array-tc.cc
+++ b/src/TEMPLATE-INST/Array-tc.cc
@@ -30,17 +30,17 @@ along with Octave; see the file COPYING.
 #include "Array.h"
 #include "Array.cc"
 
 #include "Array2.h"
 
 #include "ArrayN.h"
 #include "ArrayN.cc"
 
-#include "oct-obj.h"
+#include "ov.h"
 
 #include "oct-sort.cc"
 
 NO_INSTANTIATE_ARRAY_SORT (octave_value);
 
 INSTANTIATE_ARRAY (octave_value, OCTINTERP_API);
 
 template class OCTINTERP_API Array2<octave_value>;
